OpenRA.Game/Graphics/ModelRenderer.cs

```diff
@@ -79,7 +79,7 @@ public void SetViewportParams()
 			shader.SetMatrix("View", view);
 		}
 
-		public ModelRenderProxy RenderAsync(
+		public ModelRenderProxy Render(
```

Comment:
This isn't using any `async` code in the literal sense, but doesn't do any rendering right now either if I understand correctly (since it is appending to `doRender`). So it might make sense to keep this. 🤔

Reply:
True, reverted.